examples/sitespinner-sample.aliases.drushrc.php

```diff
@@ -0,0 +1,209 @@
+	 *
+	 * This file creates 3 aliases:
+	 * 1. "template" - points to the live, pre-existing template site from which settings are fetched for re-use
+	 * 2. "parent" - extends "template" alias while offering more granular control for creating children with similar configuration settings
+	 * 3. "destination" - you must customize this value! (e.g. "peace")
+	 *
+	 * COMPLETE INSTRUCTIONS: See https://github.com/Islandora-Collaboration-Group/drush-sitespinner
+	 *
+	 * CREATE A NEW MULTI-SITE: Specify your "template alias" and your "destination alias"
+	 *    drush sitespinner @sitespinner-sample.template @sitespinner-sample.peace
+	 *    The command above will create a new multi-site at: https://subdomain.domain.edu/peace
+	 *
+	 * DELETE A MULTI-SITE: Specify your "destination alias" (e.g "peace")
+	 *    drush sitespinner-delete @sitespinner-sample.peace
+	 *    The command above will will permanently delete the site's files (sites/{site root} directory), symlink, and corresponding MySQL database and tables
+	 *
+	 * USEFUL drush commands
+	 *    drush sa                               [list all site aliases] ["sa" is shortcut for "site-alias"]
+	 *    drush sa @sitespinner-peace            [shows execution view of a specific alias file, without any actual execution]
+	 *    drush cc drush                         [clear drush cache] ["cc" is shortcut for "cache-clear"]
+	 */
+
+	# ---------------------------
+	# Configuration Settings ("Use the Force, Luke.")
+	# ---------------------------
+
+	#--- New destination multi-site ---#
+	$DESTINATION_SITE_PRETTY_TEXT = "A peaceful new site";      # new site description
+	$DESTINATION_SITE_DIRECTORY   = "peace";                    # new site directory name; alphanumeric or dashes; no spaces or underscores; omit trailing slash "/"
+	#--- Preexisting template (source) multi-site ---#
+	$TEMPLATE_SITE_DIRECTORY = "templatesite";                  # directory of live template site to be copied; alphanumeric or dashes; no spaces or underscores; omit trailing slash "/"
+	#--- Server host names ---#
+	$DOMAIN_NAME          = "subdomain.domain.edu";             # subdomain.domain.edu; leave empty if none exists; omit "https://"; omit trailing slash "/"
+	$FEDORA_HOST_NAME     = "subdomain-fedora.domain.edu";      # full host name (including subdomain, if exists) of server running Fedora (fedora.domain.edu)
+	$DRUPAL_WEB_ROOT_PATH = "/var/www/html";                    # "root" is where the web server's index.php file is located, not the multi-site root; omit trailing slash "/"
+	$SECURITY_MODE        = "https://";                         # "https://" or "http://"
+	#--- MySQL authentication (consider granting all privileges for this db user: absolutely requires CREATE, SELECT, INSERT, DELETE, DROP) ---#
+	$MYSQL_HOST_NAME = "subdomain-webdb.domain.edu";            # full host name (include subdomain, if exists) of server running MySQL (mysql.domain.edu)
```

Comment:
You really should provide two mysql users, one for the site creator, and the other for the drupal user. The user that runs the site should not have the ability to create and destroy databases.